docs/guides/publisher-client-side.md

```diff
@@ -45,28 +51,30 @@ If you are using Google Ad Manager and want to use the secure signals feature, f
- 3. [Refresh tokens](#refresh-tokens)
- 4. [Clear Identity: user logout](#clear-identity-user-logout)
+ 1. [Establish identity: User Login](#establish-identity-user-login)
+ 2. [Bid Using UID2 Tokens](#bid-using-uid2-tokens)
+ 3. [Refresh Tokens](#refresh-tokens)
+ 4. [Clear Identity: User Logout](#clear-identity-user-logout)
 
 ### Establish Identity: User Login
 
 
-After authentication in step 1-c, which forces the user to accept the rules of engagement and allows the publisher to validate their email address or phone number, a UID2 token must be generated on the server side. The following table details the token generation steps.
+After authentication in step 1-c, which forces the user to accept the rules of engagement and allows the publisher to validate the user's email address or phone number, a UID2 token must be generated on the server side. The following table details the token generation steps.
 
 | Step | Endpoint/SDK | Description |
 | :--- | :--- | :--- |
-| 1-d | [POST /token/generate](../endpoints/post-token-generate.md) | After the user authenticates and authorizes the creation of a UID2, use the [POST /token/generate](../endpoints/post-token-generate.md) endpoint to generate a UID2 token using the provided normalized email address or phone number of the user. |
-| 1-e | [POST /token/generate](../endpoints/post-token-generate.md) | Return a UID2 token generated from the user's email address, phone number, or the respective hash. |
-| 1-f | UID2 SDK for JavaScript | Send the returned UID2 token from step 1-e to the SDK in the `identity` property of its [init() function](../sdks/client-side-identity.md#initopts-object-void). |
-| 1-g | UID2 SDK for JavaScript | Provide a callback function to the SDK which will receive identity updates and use them to initiate targeted advertising. |
+| 1-d | [POST /token/generate](../endpoints/post-token-generate.md) | After the user authenticates and authorizes the creation of a UID2, use the [POST /token/generate](../endpoints/post-token-generate.md) endpoint to generate a UID2 token using the email address or phone number provided by the user. Make sure it is normalized. |
+| 1-e | [POST /token/generate](../endpoints/post-token-generate.md) | The endpoint returns a UID2 token generated from the user's email address, phone number, or the respective hash. |
+| 1-f | UID2 SDK for JavaScript | The SDK sends the returned UID2 token from step 1-e to the SDK in the `identity` property of its [init() function](../sdks/client-side-identity.md#initopts-object-void). |
+| 1-g | UID2 SDK for JavaScript | Provide a callback function to the SDK, which will receive identity updates and use them to initiate targeted advertising. |
+
+(**GWH_LP02 Hoping to clarify 1-g above. The SDK is listed as the actor but the statement says "Provide a callback function to the SDK" -- could we word this better? Maybe something like, set up the SDK's callback function to receive...?**)
```

Comment:
Is this clearer?

> Provide the SDK a callback function which will receive identity updates and use them to initiate targeted advertising.

The intent is that they need to write the callback function (which receives identity updates and initiates targeted advertising - that's their code). The SDK provides the identity updates, but it's up to the publisher to do something with them.

Reply:
TY! Here's what I've put, LMK what you think. We can tweak it any way you want or go back to the original.
"Provide the SDK a callback function that will receive identity updates from the SDK and use them to initiate targeted advertising."